commandsummary/commandsummary.go

```diff
@@ -144,6 +201,28 @@ func ShouldRecordSummary() bool {
 	return os.Getenv(coreutils.OutputDirPathEnv) != ""
 }
 
+func createAndWriteToFile(filePath, fileName string, data []byte) (err error) {
```

Comment:
Let's add a description for this function and its arguments.
In addition, let's add comments that explain its code flow, because the logic of the below code flow isn;t quite clear - 
```
	if strings.Contains(fileName, "*") {
		fd, err = os.CreateTemp(filePath, fileName)
	} else {
		fd, err = os.Create(filepath.Join(filePath, fileName))
	}
```